api/openapi-spec/openapi.yaml

```diff
@@ -347,12 +344,12 @@ components:
           type: string
         client_secret:
           type: string
-  securitySchemes:
-    SdkKeyAuth:
-      in: header
-      name: X-Optimizely-SDK-Key
-      type: apiKey
-    TokenAuth:
-      type: http
-      scheme: bearer
-      bearerFormat: JWT
+#  securitySchemes:
```

Comment:
did you get a chance to take a look at this? We should ideally be asserting security part too.